test-harness/specs/dynamic_json_responses-AC-1.dynamic-response.oas2.txt

```diff
@@ -0,0 +1,39 @@
+====test====
+Prism generates dynamic values for schema when no static examples are provided.
+
+Given there are no examples defined for an operation
+And this operation can produce application/json response
+And this operation response defines a JSON Schema (schema field)
```

Comment:
```suggestion
And this operation response defines a Schema Object
```

Let's use OpenAPI terminology here and call it a Schema Object. It's not JSON Schema so we should not say it is.